hangman/lib/hangman/application.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Hangman.Application do
+  use Application
+
+  def start(_type, _args) do
```

Comment:
# +4

So I'm going to give you a +5 for adding a store, but then take one away for the supervision strategy. If the store dies, don't you also have to kill the game?